applications/ALEapplication/custom_elements/laplacian_meshmoving_element.cpp

```diff
@@ -55,179 +55,227 @@
 #include "utilities/geometry_utilities.h"
 #include "custom_elements/laplacian_meshmoving_element.h"
 
-namespace Kratos {
-
-LaplacianMeshMovingElement::LaplacianMeshMovingElement(
-    IndexType NewId, GeometryType::Pointer pGeometry)
-    : Element(NewId, pGeometry) {
-  mThisIntegrationMethod = GetGeometry().GetDefaultIntegrationMethod();
+namespace Kratos
+{
+LaplacianMeshMovingElement::LaplacianMeshMovingElement(IndexType NewId,
+                                                       GeometryType::Pointer pGeometry)
+    : Element(NewId, pGeometry)
+{
+    mThisIntegrationMethod = GetGeometry().GetDefaultIntegrationMethod();
```

Comment:
I'm not familiar with this part as it is from the old code (file was formatted). I will leave this part to @AndreasWinterstein to consider.